### ISSUE TITLE
bounceIn without animated class just works

### ISSUE THREAD

**Issue by ubaltaci** [animate-css/animate.css#293]
I'm not sure this is an issue or not since I'm not familier whole concept of this css animation thing :)

`el.addClass("bounceIn");` without adding `animated` class animates related dom element.

**Comment by frasern** [comment]
This is happening because `bounceIn`, `bounceOut`, `flipOutX` and `flipOutY` have a specific animation duration of 0.75s set.

`hinge` also has a specific duration of 2s set, but this has been implemented differently for some reason and only applies when the `animated` class is also present. This seems like the expected behaviour to me.

The inconsistent behaviour for those 4 classes was causing me issues, so I've forked a version that only sets the .75s duration on those classes when the  `animated` class is also present: https://github.com/frasern/animate.css/commit/3b7735b68979a0d74ca2302f14d456f8ff84c759

Happy to create a pull request if it would help.
